src/datasets/features.py

```diff
@@ -775,7 +829,16 @@ def generate_from_dict(obj: Any):
     return class_type(**{k: v for k, v in obj.items() if k in field_names})
 
 
-def generate_from_arrow_type(pa_type: pa.DataType):
+def generate_from_arrow_type(pa_type: pa.DataType) -> FeatureType:
+    """
+    generate_from_arrow_type accepts an arrow DataType and returns a datasets FeatureType to be used as the type for
```

Comment:
docstring I left while I was working locally - happy to remove or keep based on guidance from the maintainers!